### PR TITLE
Showing Pagination while reloading the page everytime

### PR DESCRIPTION
**Pull request by GangulaMounika**
## Description

Showing Pagination while reloading the page every time.
Changed Add New button color in Patient Search to make it look disabled.
Profile - Header needs more padding on left
Search - line border colors should match #DCDEE0

## Tickets

* https://cdc-nbs.atlassian.net/browse/CNFT1-1582
* https://cdc-nbs.atlassian.net/browse/CNFT1-1590
* https://cdc-nbs.atlassian.net/browse/CNFT1-1592
* https://cdc-nbs.atlassian.net/browse/CNFT1-1594

## Steps to verify

1. Patient Search- Showing Pagination while reloading the page every time.
2. Patient Search- Changed Add New button color in Patient Search to make it look disabled.
3. Patient Profile - Added more padding on left to the Header
4. Patient Search - line border colors should match #DCDEE0

### CODE REVIEW

File: apps/modernization-ui/src/pages/patient/profile/PatientProfile.tsx [hunk 1]
```diff
@@ -84,8 +84,8 @@ export const PatientProfile = () => {
     return (
         <ProfileProvider id={id}>
             <div className="height-full main-banner">
-                <div className="bg-white grid-row flex-align-center flex-justify border-bottom-style">
-                    <h1 className="font-sans-xl text-medium">Patient profile</h1>
+                <div className="padding-left-2 padding-right-1 bg-white grid-row flex-align-center flex-justify border-bottom-style">
+                    <h1 className="font-sans-xl text-medium">Patient Profile</h1>
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/pages/patient/profile/PatientProfile.tsx:88]: issue: The page title should remain sentence cased.  "Patient Profile"

File: apps/modernization-ui/src/setupProxy.js [hunk 2]
```diff
@@ -11,8 +11,21 @@ const LOGIN = function (pathname, req) {
 };
 
 module.exports = function (app) {
+    // Modernization API
     app.use(
-        createProxyMiddleware([NBS_API, GRAPHQL, ENCRYPTION, PAGEBUILDER_API], { target: 'http://localhost:8080/' })
+        createProxyMiddleware([NBS_API, GRAPHQL, ENCRYPTION, PAGEBUILDER_API], {
+            target: process.env.npm_config_backend,
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/setupProxy.js:17]: issue:  This prevents the `npm run start` from running if the environment variable is not set.  Can you include `http://localhost:8080/` as a default?

**Review by adamloup-enquizit — CHANGES_REQUESTED** [review]
I'd prefer using `onCompleted` to handle the search results over async / await but I understand that's not a straightforward change.  We will have to address this when we re-build this page.

Please correct the case for "Patient profile" and add a default value for the `NPM_CONFIG_BACKEND` envrionment variable.

File: apps/modernization-ui/src/setupProxy.js (hunk 2 above)
Comment (GangulaMounika) [reply apps/modernization-ui/src/setupProxy.js:17]: Oh no my intention is not to commit this. Reverting it back.

File: apps/modernization-ui/src/pages/patient/profile/PatientProfile.tsx (hunk 1 above)
Comment (GangulaMounika) [reply apps/modernization-ui/src/pages/patient/profile/PatientProfile.tsx]: Done

**Review by adamloup-enquizit — APPROVED** [review]

**Pull request merged**